FontraPakMain.py

```diff
@@ -73,13 +88,34 @@ def event(self, event):
         return True
 
 
+def getFontPath(path, fileType):
+    extension = fileTypesMapping.get(fileType)
```

Comment:
```suggestion
    extension = fileTypesMapping[fileType]
```
We should always receive a known file type here, right? Then better make sure we have a key for it, else the next line would fail confusingly (extension would be None)